custom_components/wyzeapi/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "wyzeapi",
-  "version": "2021.10.2",
+  "version": "2021.11.1b1",
```

Comment:
Should this be b1? The PR title is 2021.11.1

Reply:
Ahh true. I should fix that